teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
- [ ] Since the Teacher class inherits from the Person we should provide the needed parameters for creating a Person and also pass them up to the parent class:

  ```
  def initialize(age, specialization, name = "Unknown", parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @specialization = specialization
  end
  ```